### PR TITLE
Add scripts/docker_make.sh

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

### CODE REVIEW

File: scripts/docker_make.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# could not run in image based on python:3.6-buster which has older libc.
+#
+# To build an envoy binary, run
+#
+#    scripts/docker_make.sh build-envoy
+#
+
+# The latest Prow docker image used for prow build/tests
+# It is built from docker/Dockerfile-prow-env
+IMAGE=gcr.io/cloudesf-testing/gcpproxy-prow:v20200207-v2.4.0-9-g17334b8-master
```
Comment (nareddyt) [inline scripts/docker_make.sh]: Can we just use the latest version of the prow image? Otherwise, we need to remember to always update this.

Comment (qiwzhang) [reply scripts/docker_make.sh]: Done

**Pull request merged**